support/operations.md

```diff
@@ -0,0 +1,54 @@
+# Operations overview
+
+## How we think of Ops
+
+!> We don't have a tiered approach to operations and support, there is no first or second line, we are it.<br/><br/>Instead, we operate what's known as 'Swarm' support model - work with your friends to fix the problem, our users are our service so if they have a problem, we have a problem.<br/><br/>The goal is not to get rid of the ticket, it's to get rid of the problem and that means resolving the issue.
+
+Support and operations is the most important part of our service. and **It's everyone's job to support the services we build.**
+
+To make sure that we develop to the highest standards, everyone works in Ops on a cycle. This means that everyone experiences the bugs, issues, feedback and new feature requests first hand and can take that experience in to project work to ensure that new design and development does not introduce the same kind of problems.
+
+We treat Ops like a project, this way everyone gets to experience the issues that users face and their feedback. We get to feel their pain and understand the problems they encounter, first hand.
+
+?> If you're not scheduled on a project, you're working with the rest of the team supporting services
+
+## How we handle Ops
+
+As with our other projects, there's a daily stand up, measurements and an owner. Ops is no different.
+
+The owner of Ops is the Operations Lead which, for the time being, is the Digital Technolgies Manager.
+
+### I'm not on a project, what do I do?
+1. Attend the [daily call](support/ops-stand-up.md)
+1. Open Lets Go Direct (LANDesk) and follow the [guide on how we handle support tickets](support/handling-tickets.md)
+1. Pitch improvements to the Operations Lead ([see 'How we improve Ops'](#how-we-improve-ops))
+1. Remind yourself of the [two types of work](support/two-types-of-work.md)
+1. Learn something new
+
+## How we measure Ops
+!> in progress
+
+For each service we manage we measure
+
+* Cumulative flow
+  - how many tickets resolved vs created in a given period
+* Reason for failure
+* Mean time to failure (MTTF)
+  - aka “uptime”
```

Comment:
I think that, technically, MTTF is slightly different to uptime. Uptime is derived from a combination of MTTF, MTTD, and MTTR. But some people might say I'm being pedantic...

Reply:
Not pedantic at all, I have a habit of over-simplifying to the point where meaning is lost so thanks for picking up on this one.